src/dialog/reimu.c

```diff
@@ -1,7 +1,7 @@
 /*
  * This software is licensed under the terms of the MIT License.
  * See COPYING for further information.
- * ---
+ * —-
```

Comment:
Hm, maybe you could exclude the copyright notices from your substitution script somehow?

Reply:
Nevermind, I see you fixed that already.